distribution/pull_v2_test.go

```diff
@@ -129,7 +129,7 @@ func TestValidateManifest(t *testing.T) {
 	var goodSignedManifest schema1.SignedManifest
 	err = json.Unmarshal(goodManifestBytes, &goodSignedManifest)
 	if err != nil {
-		t.Fatal("error unmarshaling manifest:", err)
+		t.Fatal("error unmarshalling manifest:", err)
```

Comment:
@AkihiroSuda : I only change a word like that if it's used inconsistently, in this case, here are the preexisting instances of the alternate spelling that I arbitrarily selected:
```
CHANGELOG.md:- Fix unmarshalling of Command and Entrypoint
CHANGELOG.md:* Use same error handling while unmarshalling CMD and ENTRYPOINT
registry/endpoint_v1.go:                logrus.Debugf("Error unmarshalling the _ping PingResult: %s", err)```

I'm happy to drop the current change, but I will replace it w/ a change to these three files to go the other direction.